tools/lib/src/process_runner_apis.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:convert';
+import 'dart:io' as io;
+
+import 'package:file/file.dart';
+import 'package:flutter_plugin_tools/src/common/core.dart';
+
+import 'package:flutter_plugin_tools/src/common/process_runner.dart';
+
+/// An extension method of [ProcessRunner] to supports synchronous operations.
+extension Synchronizable on ProcessRunner {
```

Comment:
The name of the extension doesn't much matter but as I mentioned in the previous comment, `Synchronizable` has a slightly different meaning from `synchronous`.